File: src/main/java/io/github/mg138/tsbook/command/Help.java
package io.github.mg138.tsbook.command;

import io.github.mg138.tsbook.Book;

import org.bukkit.command.CommandSender;

public class Help {
    public static boolean call(CommandSender sender) {
        sender.sendMessage(Book.Companion.getCfg().translate.translate("messages.help"));
        return true;
    }
}

File: src/main/java/io/github/mg138/tsbook/command/admin/Help.kt
package io.github.mg138.tsbook.command.admin

import io.github.mg138.tsbook.Book.Companion.setting
import org.bukkit.command.CommandSender

object Help {
    fun call(sender: CommandSender): Boolean {
        sender.sendMessage(setting.translate.translate("messages.help"))
        return true
    }
}